openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -270,19 +271,19 @@ def validate_date_of_birth(self, value):
 
     def validate_phone(self, value):
         """
-        Method which validates the phone field that should contain only numbers and brackets.
+        Method which validates the phone field that should contain only numbers, brackets and white spaces.
         """
 
-        if value is not None and "_" in value:
-            raise AccountValidationError({
-                "phone": {
-                    "developer_message": "Value is not valid for field phone",
-                    'phone': [u'The phone field should contain only numbers and brackets.'],
-                    "user_message": "This value is invalid.",
-                }
-            })
+        if re.match('^[0-9()\s]+$', value) or not value:
```

Comment:
Usually, I prefer to use the shortcut `\d` instead of `[0-9]`, which is not critical. I did not understand the point that `white spaces` are desired behavior, so a number like `0                                3` is good to store in the database. I disagree. In my strict opinion, it should contain only necessary characters without any noise like white spaces.  And web client is able to handle and how how to apply a mask for phone numbers.

An the second point, please refactor the condition  `if re.match('^[0-9()\s]+$', value) or not value:`, cause `re` action apriory is heavy so following condition will be better
```
if not value or re.match('^[0-9()\s]+$', value):
```

Reply:
The condition was changed. But about storing some extra chars (like blank spaces) in the DB. I decided that we don't need to change it. It can affect some places (CSV reports) when we can win in micro-optimization but lose in UI.

Reply:
Okay, thanks for the condition, the second - up to you)